Makefile

```diff
@@ -6,3 +6,10 @@ install-pnpm: ## Install/Update pnpm package manager
 
 install: install-pnpm ## Install all dependencies
 	pnpm install
+	# Extension
+	cd extension && pnpm install
+	# Backend
+	cd backend && pnpm install
+
+build-extension:
+	cd extension && pnpm run build
```

Comment:
Il faudrait pas avoir des nested Makefiles pour que ce soit plus simple ?

Reply:
```suggestion
	# Extension
	cd extension && pnpm install
	# Backend
	cd backend && pnpm install

build-extension:
	cd extension && make run build
```

Reply:
If we were going to use nested Makefile, then `$(MAKE) -C backend install` would be the correct way to recursively call `make install` in a subdirectory.

However I worry that using Makefiles for this project is not the right approach, since it has a duplicate purpose with `package.json`.